crud/chrisModuleCRUD.java

```diff
@@ -29,5 +29,22 @@
         }
         
         return moduleNames;
-    } 
+    }
+    
+    public static int generateModuleID(Statement stment) {
+        String maxIDSql = "SELECT MAX(module_id) FROM fake_modules"; //DEV: NOT THE REAL TABLE
```

Comment:
would prefer that you have the actual table name in here, but that isn't enough really to block the PR